c++/src/capnp/compiler/capnp.c++

```diff
@@ -19,6 +19,10 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+#if __CYGWIN__
```

Comment:
You can remove the `#if __CYGWIN__` from all of these. We can define this macro on all platforms. (In fact, I think it's on by default for C++ on Linux.)

Reply:
Done!